src/features/content-insights/_variables.scss

```diff
@@ -0,0 +1,13 @@
+@use "sass:color";
+
+@import '../../styles/variables';
+
+$ActivityList-event-spaceBetween: $bdl-grid-unit * 3;
+$ActivityList-header-spaceBetween: $bdl-grid-unit * 2;
+$ActivityList-eventGroupTitle-spaceAbove: $bdl-grid-unit * 4;
```

Comment:
what are these used for? is `ActivityList` in a separate PR?

Reply:
I think SUIT naming would use the property name instead of the value: https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#variables

so `$ActivityList-event-justifyContent` although I don't think our repo currently follows a strict pattern for SCSS variables.

Reply:
Good catch, I think this can be removed